keylime/cloud_verifier_common.py

```diff
@@ -166,9 +166,9 @@ def process_quote_response(agent, json_response, agentAttestState) -> Failure:
     failure.merge(quote_validation_failure)
 
     if not failure:
-        # set a flag so that we know that the agent was verified once.
-        # we only issue notifications for agents that were at some point good
-        agent["first_verified"] = True
+        agent["attestations"] += 1
```

Comment:
Overall really like the change, but I wonder about the field name. I think `attestation_count` would be a better name since it won't be mistaken for being a list of attestation objects or something similar.

Reply:
fair enough, will make the change